tool.go

```diff
@@ -636,12 +636,14 @@ func (fs serveCommandFileSystem) Open(requestName string) (http.File, error) {
 		if err == nil {
 			return f, nil
 		}
+		defer f.Close()
```

Comment:
Yes, @neelance, it doesn't have to be, but it could be (and most likely is). See my review comments above.